src/main/flight/pid.c

```diff
@@ -112,6 +112,8 @@ PG_RESET_TEMPLATE(pidConfig_t, pidConfig,
 
 #define CRASH_RECOVERY_DETECTION_DELAY_US 1000000  // 1 second delay before crash recovery detection is active after entering a self-level mode
 
+#define LAUNCH_CONTROL_YAW_ITERM_LIMIT 50 // yaw iterm windup limit when launch mode is "FULL" (all axes)
```

Comment:
Maybe it should be configurable in the future?

Reply:
@borisbstyle If it becomes an issue in testing during the RC phase then sure we can do that. I reality I've found the "FULL" mode where yaw is also stabilized is of very limited value because yaw winds up so easily. In fact I originally had yaw support, took it out, then finally added it back before making the PR. I did a bunch of testing and found that once yaw I reached about > 70 we started to run the risk of windup. Around 100 and it pretty much got out of control. So I didn't add the parameter now because there's not really much of a tuning window and to avoid having more things to confuse the user.